src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,25 @@
-import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions } from '../../features/filter';
+import { Status } from '../../types/Status';
+
+export const TodoFilter = () => {
+  const dispatch = useAppDispatch();
+  const { status, query } = useAppSelector(state => state.filter);
 
-export const TodoFilter: React.FC = () => {
   return (
-    <form
-      className="field has-addons"
-      onSubmit={event => event.preventDefault()}
-    >
+    <form className="field has-addons">
```

Comment:
You should prevent the default form submission event to avoid page reload. Add 'onSubmit' event to the form and call 'event.preventDefault()' inside.